zboss/include/osif/zb_revision_r22.h

```diff
@@ -29,7 +29,7 @@
  *
  * Note: This value is automatically updated by the release script.
  */
-#define ZBOSS_PLATFORM_PATCH 0U
+#define ZBOSS_PLATFORM_PATCH 1U
```

Comment:
What is the reason for having the PATCH value set to 1?

Reply:
Not sure, will check this

Reply:
Patch value kept at 0, platform minor version set to 1U due to changes with NCP prot version